src/Maker/MakeDockerDatabase.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\Docker\DatabaseServices;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Str;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ *
+ * @internal
+ */
+final class MakeDockerDatabase extends AbstractDockerMaker
+{
+    /**
+     * @var string
+     */
+    private $databaseChoice = '';
+
+    public static function getCommandName(): string
+    {
+        return 'make:docker:database';
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConfig): void
+    {
+        $command
+            ->setDescription('Create a database Docker container.')
+        ;
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command): void
+    {
+        parent::interact($input, $io, $command);
+
+        $io->newLine();
+
+        $this->databaseChoice = $io->choice(
+            'Which database service will you be creating?',
+            ['MySQL', 'MariaDB', 'Postgres']
+        );
+
+        $this->serviceName = strtolower($this->databaseChoice);
+
+        $io->text([sprintf(
+            'For a list of supported versions, check out https://hub.docker.com/_/%s',
+            $this->serviceName
+        )]);
+
+        $this->serviceVersion = $io->ask('What version would you like to use?', 'latest');
+
+        if ($this->composeFileManipulator->serviceExists($this->serviceName)) {
+            $this->serviceAlreadyDefinedQuestion($io);
+
+            $this->serviceName = $io->ask(sprintf(
+                'What name should we call the new %s service? e.g. %s',
+                $this->serviceName,
+                str_replace(' ', '-', Str::getRandomTerm())
+            ));
```

Comment:
Actually, we need to be opinionated here. If the service is called `database`, then, iirc, the Symfony binary will expose `DATABASE_URL`. So, we should suggest `database` as the default. We should maybe even print a note about how the name will map to the exposed env var with the Symfony binary... but I'm not positive on that